### DISCUSSION TITLE
what happens to files in the annex when you delete them?

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#372] [Q&A] [answered]
I made a commit with a bunch of audio files that I think are duplicates of other audio files:

C30/C30_20180717/C30_20180717_converted/REC001.WAV
versus
C30/C30_20180717/C30_20180717/REC001.WAV

I guess I could leave them in with no reference to them from `metadata/recordings.csv`, but I may find this confusing in the future. For now, I'm adding a column to `metadata/recordings.csv` called "repeated" that is 1 if I think these are duplicates.

If/when I decide to delete the duplicate audio files, will I be able to go back to a different commit and retrieve them? Probably not, right?

**Comment by alecristia** [comment, marked as answer]
you cannot delete them unwittingly, see http://handbook.datalad.org/en/0.12/basics/101-136-filesystem.html#removing-annexed-content-entirely

**Comment by LoannPeurey** [comment]
I would have to guess that you won't be able to retrieve them. Going back
to the commit would retrieve symbolic links but when trying to get them in
the annex, they should not be stored there anymore.

I should run a test of this at some point just to be sure.


Le lun. 25 avr. 2022 à 09:41, Alex Cristia ***@***.***> a
écrit :

**Comment by LoannPeurey** [comment]
Indeed, But you can delete them with datalad drop (that is the command you
would use for the original question)

Le lun. 25 avr. 2022 à 09:45, Alex Cristia ***@***.***> a
écrit :
